cookbooks/cdo-varnish/templates/default/default.vcl.erb

```diff
@@ -578,6 +586,22 @@ sub vcl_deliver {
   set resp.http.X-Varnish-Node = "<%= node.name %>";
 }
 
+sub vcl_hash {
+  hash_data(req.url);
+  if (req.http.host) {
+    hash_data(req.http.host);
+  } else {
+    hash_data(server.ip);
+  }
```

Comment:
The above is duplicated from the default [vcl_hash](https://www.varnish-software.com/static/book/VCL_functions.html#vcl-vcl-hash) function, the default logic adds the hostname (or IP) and the URL to the cache hash